blosc2/blosc2_ext.pyx

```diff
@@ -1981,6 +1993,14 @@ cdef check_schunk_params(blosc2_schunk* schunk, kwargs):
             raise ValueError("Cannot change typesize with this mode")
 
 
+cdef schunk_is_ndarray(blosc2_schunk* schunk):
+    meta1 = "b2nd"
+    meta1 = meta1.encode("utf-8") if isinstance(meta1, str) else meta1
+    meta2 = "caterva"
```

Comment:
`caterva` layer is pre-2.0.  The next release of python-blosc2 will be 3.0 (beta), so this is perfect time to get rid of this check.

Reply:
Note: I also removed the `test_persistency_caterva` test to make this work. The file `persistency.cat` was probably created pre-2.0.